Tools/build/check_warnings.py

```diff
@@ -49,9 +49,7 @@ def extract_warnings_from_compiler_output_json(
     """
     # Regex to find json arrays at the top level of the file
     # in the compiler output
-    json_arrays = re.findall(
-        r"\[(?:[^[\]]|\[[^\]]*\])*\]", compiler_output
-    )
+    json_arrays = re.findall(r"\[(?:[^[\]]|\[[^\]]*\])*\]", compiler_output)
```

Comment:
Does this work? My IDE says they were redundant escapes:
```suggestion
    json_arrays = re.findall(r"\[(?:[^[\]]|\[[^]]*])*]", compiler_output)
```

Reply:
Removed redundant escapes